lib/iris/tests/results/unit/analysis/cartography/project/TestAll/cube.cml

```diff
@@ -40,23 +37,20 @@
         </dimCoord>
       </coord>
       <coord datadims="[2, 3]">
-        <auxCoord id="81715b9b" points="[[165.651442753, 178.244991596, -168.158926231,
-		 ..., 169.237783452, -176.909512275,
-		 -164.826658657],
-		[171.94140771, -164.537718135, -145.687720691,
-		 ..., 147.432981219, 167.102019536,
-		 -169.282739452],
-		[-175.009064548, -147.024893924, -126.259032068,
-		 ..., 128.51309535, 150.23087257, 178.899398957],
+        <auxCoord id="81715b9b" points="[[inf, inf, -168.158920348, ..., 169.237777806,
+		 inf, inf],
+		[inf, -164.537714569, -145.687716669, ...,
+		 147.432977295, 167.102016129, inf],
+		[-175.009062798, -147.024891707, -126.259029624,
+		 ..., 128.513092975, 150.23087042, 178.899397359],
 		..., 
-		[-58.3498044086, -70.8813751857, -58.2031018979,
-		 ..., 60.1459626823, 72.0019743312, 20.695582309],
-		[22.0508714127, -35.1608425218, -42.536925264,
-		 ..., 43.0527802786, 31.4703414616,
-		 -27.6376083083],
-		[17.2456810934, 2.60343563417, -15.3083834763,
-		 ..., 14.270914103, -4.55555121054,
-		 -17.7002310265]]" shape="(10, 10)" standard_name="longitude" units="Unit('degrees')" value_type="float64">
+		[-58.34983089, -70.8813785605, -58.2031036304,
+		 ..., 60.1459645527, 72.0019781785,
+		 20.6956240803],
+		[inf, -35.1608514733, -42.5369291003, ...,
+		 43.052784475, 31.4703511243, inf],
+		[inf, inf, -15.3083916055, ..., 14.2709222711,
+		 inf, inf]]" shape="(10, 10)" standard_name="longitude" units="Unit('degrees')" value_type="float64">
```

Comment:
New `inf` values are an improvement for points outside the Robinson projection - they replace spurious 'nonsense numbers' (https://github.com/OSGeo/PROJ/pull/2151). It is unlikely that this change has negative downstream consequences - @stephenworsley has confirmed the data at these points is already masked and that basic metrics (e.g. mean, stdev) upon regridding are unaffected 👍